src/css/App.css

```diff
@@ -4,43 +4,193 @@ a {
 
 .centre-text {
   text-align: center;
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 2rem;
+}
+
+.content {
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+.content a {
+  color: #fff;
+}
+
+.content a:hover {
+  color: #fd79a8;
+}
+
+.left-text {
+  text-align: left;
+  width: 80% !important;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 2rem;
+}
+
+.gap-div {
+  height: 4rem;
 }
 
 .Menu a {
   color: #fff;
+  text-align: right !important;
 }
 
 .Menu a:hover {
   color: #fd79a8;
 }
 
+.navbar {
+  align-items: right;
+  text-align: right !important;
+}
+
+.sosBtn a {
+  background-color: #fd79a8;
+  color: #343a40!important;
+  font-weight: bold;
+  border: 1px solid #fff;
+  border-radius: 5px;
+  padding: 10px;
+}
+
+.sosBtn a:hover {
+  background-color: #fff;
+  color: #fd79a8 !important;
+  font-weight: bold;
+  border: 1px solid #fd79a8;
+  border-radius: 5px;
+  padding: 10px;
+}
+
+.highlightNav a {
+  border: 1px solid #fff;
+  border-radius: 5px;
+  padding: 10px;
+}
+
+.highlightNav a:hover {
+  border: 1px solid #fd79a8;
+  border-radius: 5px;
+  padding: 10px;
+}
+
+.nav-item {
+  padding: 10px;
+}
+
+.nav-tabs .nav-link {
+  background-color: #343a40;
+}
+
+
+.Menu .navbar-collapse {
+  margin-top: 20px;
+  margin-left: auto;
+  align-items: right;
+  text-align: right !important;
+}
+
 .App-logo {
   height: 40vmin;
   pointer-events: none;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.Header {
-  background-color: #282c34;
-  min-height: 100vh;
+.BigHeader {
+  background-color: #343a40;
+  min-height: 50vh;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.SmallHeader {
+  background-color: #343a40;
+  min-height: 140px;
+  display: flex;
+  flex-direction: column;
+}
+
+.SmallHeader h1 {
+  color: #fd79a8;
+  font-size: 30pt;
+  font-weight: bold;
+  text-align: center;
+  margin-top: 6rem;
+}
+
+.BigHeader h1 {
+  font-size: calc(6vw + 2rem);
+  color: #fd79a8;
+  font-weight: bold;
+  text-align: center;
+  margin-top: 8rem;
+}
+
+.BigHeader p {
+  font-size: calc(1vw + 1rem);
+  margin-top: 60px;
+  color: #fff;
+}
+
+.SectionHeader {
```

Comment:
any way we can integrate SCSS or SASS into this so we don't have messy af css?

Reply:
I know that by default vue's styling is in SASS, looks like there's a way to integrate: 

https://hugogiraudel.com/2015/06/18/styling-react-components-in-sass/

Reply:
I can look into using sass, I have never used sass before personally which is why I didnt bother. Again, this is just the prototyping phase for the layout of the site, so I didn't think of it.

An alternative I considered was different css files for different pages for better organization. 

Also, I want to make the point that large css files are extremely common for large sites, and just something most developers deal with if css is what they are familiar with. I think this css file is fairly well-organized for its size. css always appears "messy". If you think this is "messy", you should see the css at most production companies lol